ports/libmlservo/src/lib.rs

```diff
@@ -162,6 +195,8 @@ pub unsafe extern "C" fn heartbeat_servo(servo: *mut ServoInstance) {
                         }
                     }
                 },
+                EmbedderMsg::ShowIME(..) => (servo.keyboard.0)(servo.app, true),
```

Comment:
ShowIME has an argument that can be used to hint more specific kinds of keyboards to display. Want to file a followup to make use of that information by modifying the fields of KeyboardProperties appropriately?

Reply:
Filed #22523.